packages/card-support/package.json

```diff
@@ -45,10 +47,15 @@
     "@opensrp/server-service": "^0.0.13",
     "@opensrp/location-management": "^0.0.7",
     "@opensrp/notifications": "^0.0.2",
-    "papaparse": "^5.3.0"
+    "@opensrp/pkg-config": "^0.0.1",
+    "papaparse": "^5.3.0",
+    "i18next": "^19.8.4",
+    "i18next-conv": "^10.1.0",
+    "i18next-scanner": "^3.0.0"
```

Comment:
@kelvin-muchiri should this go to devDependencies.
how are we using i18next-conv? If I was the one that added it, it is probably not being used anymore.